backend/src/core/services/template.service.ts

```diff
@@ -1,15 +1,21 @@
 import { v4 as uuid } from 'uuid'
+import { difference, keys } from 'lodash'
+
 import S3 from 'aws-sdk/clients/s3'
 import { Transaction } from 'sequelize/types'
 
 import config from '@core/config'
 import logger from '@core/logger'
+import { isSuperSet } from '@core/utils'
+import { MissingTemplateKeysError } from '@core/errors/template.errors'
 import { configureEndpoint } from '@core/utils/aws-endpoint'
 import { jwtUtils } from '@core/utils/jwt'
 import { Campaign } from '@core/models'
 import { CsvStatusInterface } from '@core/interfaces'
 
 const MAX_PROCESSING_TIME = config.get('csvProcessingTimeout')
+type CSVParams = { [key: string]: string }
```

Comment:
Can this type be imported from the `postman-templating` module instead of redeclaring again here?

Reply:
Yup it can. I will make another PR after this one is in that replaces the `templateClient`, `CSVParams`, the errors types, as well as uninstalling the libraries that are no longer in use for worker and backend.